src/ExternalAPI/API.cs

```diff
@@ -117,7 +126,79 @@ public static string PlaceStatic(string modelName, string bodyName, double lat,
             return null;
         }
 
+        #region groups
 
+        public static bool CreateGroup(string groupName)
+        {
+            if (!StaticDatabase.HasGroupCenter(groupName))
+            {
+                GroupCenter groupCenter = new GroupCenter
+                {
+                    RadialPosition = FlightGlobals.currentMainBody.transform.InverseTransformPoint(FlightGlobals.ActiveVessel.transform.position),
+                    Group = groupName,
+                    CelestialBody = FlightGlobals.currentMainBody
+                };
+                groupCenter.Spawn();
+                KSPLog.print(groupCenter.Group);
+                return true;
+            }
+            return false;
+        }
+
+        public static bool RemoveGroup(string groupName, string BodyName = null)
+        {
+            if (BodyName == null) { BodyName = StaticDatabase.lastActiveBody.name; }
+            groupName = BodyName + "_" + groupName;
```

Comment:
Use string interpolation when concatenating 3+ strings. There's more of such cases below.

Reply:
I have just copied this from another method and not really thought about it so thanks for letting me know. Maybe I'll look over the other classes and change it there too.